mscore/instrwidget.cpp

```diff
@@ -47,7 +47,7 @@ int StaffListItem::customTablatureIdx;
 //---------------------------------------------------------
 
 StaffListItem::StaffListItem(PartListItem* li)
-   : QTreeWidgetItem(li, STAFF_LIST_ITEM)
+      : QTreeWidgetItem(li, STAFF_LIST_ITEM)
```

Comment:
Continuing the indentation topic, changing these indents is also probably not needed, 3-spaces indents for initializer lists are commonly used in MuseScore, like in this case:
https://github.com/musescore/MuseScore/blob/eafe2e93683476e1fc719843201e8994d14c16d9/mscore/musescore.cpp#L976-L977

Reply:
How are these indents done? Are you manually indenting these? I just applied the code style schema as provided in the handbook, which triggered this change. It seems like it does not allow for 'half indents' in selected cases. same for the other indenting issue, it was caused by just applying the code style schema...

Reply:
Then this schema seems to not produce always correct results. Yes, I personally do these 3-space indents manually.

Reply:
This probably should be added as a note next to the given schema in the handbook.